src/metabase/email/warn_deprecate_pulse.mustache

```diff
@@ -0,0 +1,16 @@
+{{> metabase/email/_header }}
+<p>Hi {{userName}},</p>
+<p>We deprecated Pulses in Metabase 0.38, two years ago.</p>
+<p>Now the time has come to remove them.</p>
+<p>On the next release, we will delete all Pulses and keep only Dashboard Subscriptions.</p>
+<p>These are the active Pulses you have:</p>
+<ul>
+  {{#pulses}}
+    <li><a href="{{url}}">{{name}}</a></li>
+  {{/pulses}}
+</ul>
+<p>You can transform legacy Pulses into Dashboard Subscriptions following [this doc](link-to-doc)</p>
```

Comment:
the link-to-doc will need to be filled later after Product provide me one.

Reply:
PR in flight: https://github.com/metabase/metabase/pull/36415